x/checkpointing/client/cli/utils.go

```
@@ -155,20 +157,40 @@ func newBuildCreateValidatorMsg(clientCtx client.Context, txf tx.Factory, fs *fl
 func buildWrappedCreateValidatorMsg(clientCtx client.Context, txf tx.Factory, fs *flag.FlagSet, val validator, valAc address.Codec) (tx.Factory, *types.MsgWrappedCreateValidator, error) {
 	txf, msg, err := newBuildCreateValidatorMsg(clientCtx, txf, fs, val, valAc)
 	if err != nil {
-		return txf, nil, err
+		return txf, nil, fmt.Errorf("failed to build create validator message: %w", err)
 	}
 
-	home, _ := fs.GetString(flags.FlagHome)
-	valKey, err := getValKeyFromFile(home)
-	if err != nil {
-		return txf, nil, err
+	var blsPK *bls12381.PublicKey
+	var pop *types.ProofOfPossession
+
+	blsPopFilePath, _ := fs.GetString(FlagBlsPopFilePath)
```

the error was not handled?

---

@gitferry
The error returned by `GetString` occurs only when the assigned flag value is not of type `string`. Therefore, other parts of the code using `GetString` also do not handle this error.